### PR TITLE
Use a more complete example in the README

### PR DESCRIPTION
**Pull request by Jalle19**
Generally boxes are named "organization/box-name". The name used in the manifest file must match this, otherwise Vagrant will complain about a name mismatch and you won't be able to vagrant up it. By being consistent and avoiding "fancy" directory structures it can all be automagic.

This also shows the corresponding Vagrantfile usage.

@lmars I would personally opt for removing some of the configuration items in favor of simply requiring the following:

* `basePath` (`vagrant` in the current examples), i.e. the path inside the bucket where the registry will be created
* `boxName` (`my-organization/my-cool-project` in the current examples), i.e. the full box name

Furthermore, the tool I mentioned in #25 would be easier to make if certain assumptions could be made regarding the file structure of the registry, but I guess that's moot until I actually get around to making the tool.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,42 +41,33 @@ Add the post-processor to your packer template **after** the `vagrant` post-proc
...
+        "manifest": "vagrant/json/{{ user `box_organization` }}/{{ user `box_name` }}.json",
+        "box_dir":  "vagrant/boxes/{{ user `box_organization` }}/{{ user `box_name` }}",
+        "box_name": "{{ user `box_organization` }}{{ user `box_name` }}",
         "version":  "{{ user `version` }}"
       }
     ]
   ]
 }
 ```
-**NOTE:** The post-processors must be a **nested array** (i.e.: a Packer sequence definition) so that they run in order. See the [Packer template documentation](http://www.packer.io/docs/templates/post-processors.html) for more information.
```
Comment (Jalle19) [inline README.md]: Let me know if you want me to undrop this. I figured that if someone is so familiar with Packer that they've started hunting for custom post-processors they surely know the syntax too.

Comment (lmars) [inline README.md]: I think we should leave this in (it was originally added as users didn't realise it needed to be a nested array).

**Comment by Jalle19** [comment]
@lmars I re-added the note about the configuration syntax

**Comment by lmars** [comment]
Thanks!

**Pull request merged**